content/international-candidates.md

```diff
@@ -52,217 +52,162 @@ keywords:
   - UK Ancestry Visa
 ---
 
-## Teaching or training to teach in England
+Find out how to train to become a teacher in England.
 
-Your route to working or training as a teacher in England and the financial support you might get depends on:
+[Already a teacher? Come to England to teach](/international-candidates#already-a-teacher-teach-in-england).
```

Comment:
Content wise this is fine but on the page it feels a bit odd being squished between the hero and the following `h2`:

![Screenshot from 2021-04-09 11-52-35](https://user-images.githubusercontent.com/128088/114170040-3f9f1b00-992a-11eb-81aa-4a262f73ddfd.png)

Could this be another CTA in the right column perhaps?